superflore/generators/bitbake/run.py

```diff
@@ -81,9 +81,8 @@ def main():
 
     # clone current repo
     selected_targets = active_distros
-    # try:
-    link_existing_files(args.ros_distro)
-    """
+    try:
+        link_existing_files(args.ros_distro)
```

Comment:
This goes with the ability to reuse an exisiting repo. I'd suggest `--output-repository-path` as an argument. Where if it's unset it will use a temporary directory. (If it uses a temporary directory it should delete it at the end after opening the PR) There could be a flag to prevent the cleanup. and logic to prevent you from running without a PR or preventing cleanup.